### PR TITLE
Add env vars to toggle specific crons

### PR DESCRIPTION
**Pull request by zashton**
### Test Plan 

1. add the env vars
```
CRON_NOMINEE_ENABLED=false
CRON_DAILY_UPDATE=false
CRON_EPOCH_NOTIF=false
CRON_EPOCH_CLOSE=true
```
2. trigger the cron job manually by running using its signature e.g `php artisan check:nominees`
3. check if env var is correctly blocking the cron job

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -32,3 +32,8 @@ TEST_DISCORD_WEBHOOK=
 HCAPTCHA_SECRET=
 HCAPTCHA_SITEKEY=
 COORDINAPE_USER_ADDRESS=0xfad763da9051953fea58c2304395719a3b6ba361
+
+CRON_NOMINEE_ENABLED=false
+CRON_DAILY_UPDATE=false
+CRON_EPOCH_NOTIF=false
+CRON_EPOCH_CLOSE=false
```
Comment (topocount) [inline .env.example:39]: don't we want these set to "true" for now?

**Review by topocount — COMMENTED** [review]
Soft nit: Normally, I'd request that these `signature` strings be turned into exportable constants at the root of these command files and imported in the config/cron file, but:
a. I'm not fluent enough in php to know if that's idiomatic.
b. This is part of a well-known deprecation path for this code, so my opinion isn't particularly strong if this is a good suggestion.

File: .env.example [hunk 2]
```diff
@@ -32,3 +32,8 @@ TEST_DISCORD_WEBHOOK=
 HCAPTCHA_SECRET=
 HCAPTCHA_SITEKEY=
 COORDINAPE_USER_ADDRESS=0xfad763da9051953fea58c2304395719a3b6ba361
+
```
Comment (crabsinger) [inline .env.example:35]: can we just add a comment saying `# set these to true to enable, false to disable` since some of these may be amgiguous

**Review by crabsinger — APPROVED** [review]
lgtm. maybe add a comment, but since its a deprecation path it doesnt matter much.

**Pull request merged**